cloud/README.md

```diff
@@ -0,0 +1,17 @@
+# CDK project for Prompt Injection
+
+This project uses AWS CDK (using TypeScript) to define CloudFormation templates for remote deployment of all resources
+for the Prompt Injection application.
+
+The architecture is a typical containerized node-express API managed by AWS Fargate and ECS, with load-balancing (ELB),
+plus an S3-hosted UI served through CloudFront, all secured using Cognito.
+
+The `cdk.json` file tells the CDK Toolkit how to execute your app.
+
+## Commands
+
```

Comment:
worth adding a note about signing into aws account first? Or will any cloud person find this obvious?

Reply:
Well, it depends how the user is going to authenticate with AWS. They may still be using the long-lived token way of authenticating (even though that is discouraged) and in that case they'd not need to login. As this code is currently planned to make its way into the dev branch, and therefore be part of the open-source offering, I'm inclined to let the user manage that side of things.